### PR TITLE
Move charts to keda/charts

### LINKED ISSUE #104: Deploy helm charts to a repository
Currently, we just instruct people to install from source. It would be nicer and more convenient to deploy charts to a helm repo so that we can instruct people to install without making them check out the code first.

### Use-Case

Instead of requiring folks to `git clone https://github.com/kedacore/http-add-on.git` before they run a `helm install`, I'd like folks to be able to run a `helm repo add` and then `helm install`. This allows users and operators to use the add on without interacting with the code at all.

We may consider putting charts inside of https://github.com/kedacore/charts, or hosting them with [artifact hub](https://artifacthub.io)

### Specification

- [ ] Install instructions should look similar to the ones in the [keda core ones](https://keda.sh/docs/2.1/deploy/#install)

### PR DESCRIPTION
**Pull request by khaosdoctor**
Moving charts from this repo to the official [kedacore/charts](https://github.com/kedacore/charts) repo
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] A PR is opened to update the documentation on [our docs repo](https://github.com/kedacore/keda-docs)

Fixes #104

### CODE REVIEW

**Comment by khaosdoctor** [comment]
@arschles @tomkerkhove I took the time to do this housekeeping tasks today, can you take a look at this? 

PR is https://github.com/kedacore/charts/pull/137

File: docs/install.md
Comment (tomkerkhove) [inline docs/install.md:33]: Let's update with the new name, once merged

**Review by tomkerkhove — APPROVED** [review]

File: docs/install.md
Comment (arschles) [inline docs/install.md:72]: @khaosdoctor small nit: the new spacing doesn't look lined up still. Can you adjust?

**Review by arschles — APPROVED** [review]
@khaosdoctor is there a corresponding PR in `kedacore/charts`?

File: docs/install.md
Comment (khaosdoctor) [reply docs/install.md:72]: Sure! Will fix it right away

**Review by arschles — CHANGES_REQUESTED** [review]
@khaosdoctor another thing (sorry I forgot this the first time!) - since it's just for an example walkthrough, let's leave the xkcd chart in here, and move it to `examples/`

File: docs/install.md
Comment (khaosdoctor) [reply docs/install.md]: I changed this name on the charts as well and in the latest commit

**Comment by khaosdoctor** [comment]
Also @arschles could you please fix the conflicts, I don't have permissions

File: docs/install.md
Comment (arschles) [inline docs/install.md:38]: @khaosdoctor let's follow keda's lead and not ship this repository with the chart built into it. this will keep kedacore/charts the canonical and exclusive source for the http-add-on charts, and we can remove this entire section.

**Review by arschles — CHANGES_REQUESTED** [review]
@khaosdoctor just one request. otherwise looks great, thanks :smile:

File: docs/install.md
Comment (khaosdoctor) [reply docs/install.md:38]: Moved, I think you'll need to close #150 to avoid conflicts, I already moved your changes here on 2672e17

Comment (arschles) [reply docs/install.md:38]: @khaosdoctor great, thank you! I closed #150 just now.

**Review by arschles — COMMENTED** [review]
@khaosdoctor this looks great! thanks for sticking with it 🚀 

let's wait for https://github.com/kedacore/charts/pull/137 to be merged before merging this one

**Comment by tomkerkhove** [comment]
They are merged on kedacore/charts. Feel free to remove them in this repo.

**Comment by khaosdoctor** [comment]
@arschles I think this one got stuck

**Review by arschles — APPROVED** [review]

**Comment by arschles** [comment]
@khaosdoctor CI is running hopefully one last time 😄

**Pull request merged**
